mods/ts/rules/nod-vehicles.yaml

```diff
@@ -175,6 +175,8 @@ TTNK:
 		RequiresCondition: undeployed
 	RevealOnFire:
 		ArmamentNames: primary, deployed
+	RenderVoxels:
+		Scale: 11.5
```

Comment:
Fine with me, though I guess we should compare some of the other voxels via screenshots, maybe 11.5 gives the best result for them, too.